src/presets/netlify.ts

```diff
@@ -37,27 +57,6 @@ export const netlify = defineNitroPreset({
           "utf8"
         );
       }
-
-      const functionConfig = {
-        config: { nodeModuleFormat: "esm" },
-        version: 1,
-      };
```

Comment:
Is v2 fully compatible with v1 for seamless migration of users?

I would prefer to keep v1 and incrementally add v2 implementation to allow opt-in for all new users unless netlify platform is not supporting v1 format anymore